README.md

```diff
@@ -109,7 +126,7 @@ Write this grammar using EBNF with common extensions
 
 ## A2.3
 
-«replace this with your answer»
+FSM.jpeg
```

Comment:
# 6

(you use open and closed quotes in writing. I love it!)

Reply:
Do mean writing "FSM.jpeg"?

Reply:
No: in the diagram